src/Game.py

```diff
@@ -19,10 +19,12 @@ def initGame(config1, config2, size):
         PLAYER_1: {
             "primary": getBoardFromConfig(player1Config),
             "opponent": initBoard(10),
+            "life": FLEET_LIFE
```

Comment:
Interesting to offer the posibility to make different life number for each player